frontend/src/routes/manage/Realm/Content/Edit/EditMode/Series.tsx

```diff
@@ -101,17 +101,19 @@ export const EditSeriesBlock: React.FC<EditSeriesBlockProps> = ({ block: blockRe
             css={{
                 display: "flex",
                 flexDirection: "row",
+                flexWrap: "wrap",
                 marginTop: 12,
-                marginRight: 48,
-                justifyContent: "space-between",
-                maxWidth: 400,
+                justifyContent: "start",
                 [screenWidthAtMost(BREAKPOINT_SMALL)]: {
                     flexDirection: "column",
                     gap: 12,
                 },
             }}
         >
-            <div>
+            <div css={{
+                marginRight: 36,
+                marginBottom: 12,
+            }}>
```

Comment:
Rather than that, use `row-gap` and `column-gap` on the parent element. And I'd also prefer slightly space between the items. Sth like 48 and 24?

Reply:
> And I'd also prefer slightly space between the items. Sth like 48 and 24?

Which items do you mean?

Reply:
Oh I meant the three different groups, i.e. increase the `gap` values. But no worries, we can just do it later. It's just a minor visual thing.